resources/views/auth/register.blade.php

```diff
@@ -0,0 +1,98 @@
+            </div>
+            <div>
+                @error('house_number')
+                    <div class="alert alert-danger p-2">{{ $message }}</div>
+                @enderror
+                @error('postal_code')
+                    <div class="alert alert-danger p-2">{{ $message }}</div>
+                @enderror
+            </div>
+            <div class="form-group mb-4">
+                <label for="email" class="mb-2 text-uppercase fw-bold">E-mail:</label>
+                <input type="email" class="form-control border-black w-50" id="email" name="email" value="{{old('email')}}"  placeholder="E-mail">
+                @error('email')
+                    <div class="alert alert-danger mt-1 p-2">{{ $message }}</div>
+                @enderror
+            </div>
+            <div class="form-group mb-3">
+                <label for="password" class="mb-2 text-uppercase fw-bold">Wachtwoord:</label>
+                <input type="password" class="form-control border-black w-50" id="password" name="password"  placeholder="Wachtwoord">
+                @error('password')
+                    <div class="alert alert-danger mt-2 p-2">{{ $message }}</div>
+                @enderror
+                <label for="password_confirmation" class="mb-2 text-uppercase fw-bold mt-2">Wachtwoord bevestigen:</label>
+                <input type="password" class="form-control border-black w-50" id="password_confirmation" name="password_confirmation"  placeholder="Wachtwoord">
+                @error("password_confirmation")
+                    <div class="alert alert-danger mt-2 p-2">{{ $message }}</div>
+                @enderror
+            </div>
+            <div>
+                <small>
+                    * Hieronder kies of je een particulier of zakelijke adverteerder bent.
+                </small>
+            </div>
+            <div class="form-check mt-1">
+                <input class="form-check-input" type="radio" name="type_user" id="type_user" required checked>
+                <label class="form-check-label" for="type_user">
+                    Geen adverteerder
+                </label>
+            </div>
+            <div class="form-check mt-1">
+                <input class="form-check-input" type="radio" name="type_user" id="type_user" value="particuliere adverteerder">
+                <label class="form-check-label" for="type_user">
+                    Particuliere adverteerder
+                </label>
+            </div>
+            <div class="form-check mb-4">
+                <input class="form-check-input" type="radio" name="type_user" id="type_user" value="zakelijke adverteerder">
+                <label class="form-check-label" for="type_user">
+                    Zakelijke adverteerder
+                </label>
+            </div>
```

Comment:
Id moet uniek zijn.

Ook Als je de ids uniek maakt en het for attribuut van de label het id geeft kan je klikken op de lables om te selecteren.